packages/fluentui/react-northstar/test/specs/components/Carousel/Carousel-test.tsx

```diff
@@ -266,5 +266,12 @@ describe('Carousel', () => {
       expect(wrapper.state('activeIndex')).toEqual(1);
       expect(document.activeElement.firstElementChild.innerHTML).toEqual('item2');
     });
+
+    it('should show no pagination if getItemPositionText is not passed', () => {
```

Comment:
Don't we have screener tests for this? Just wondering why you decided with unit test, when we are testing similar things with screener? Also do we have a test for the opposite scenario?

Reply:
1. Yes we have, although that depends on whether we are keeping the example with pagination in the docsite. I would rather have a unit test for this as well, we are making sure this one stays here and tests this case, and it gives faster feedback comparing it to the screener test.
2. Yep, it's `should show pagination if navigation prop is not passed` .